doc/ot_api_doc.h

```diff
@@ -59,8 +59,15 @@
  * @defgroup api-ip6                  IPv6
  * @defgroup api-srp                  SRP
  * @defgroup api-ping-sender          Ping Sender
+ * @defgroup api-tcp-group
```

Comment:
I'd add a blank row before this one for readability

Reply:
Done.